### DISCUSSION TITLE
Error occured while updating ohmyzsh

### DISCUSSION THREAD

**Discussion by nukemiko** [ohmyzsh/ohmyzsh#10317] [Help] [answered]
```
> omz update                                                       
Updating Oh My Zsh
来自 https://github.com/ohmyzsh/ohmyzsh
 * branch              master     -> FETCH_HEAD
更新 735808f4..4b1f6d63
error: 您对下列文件的本地修改将被合并操作覆盖：
        plugins/emacs/emacsclient.sh
        tools/install.sh
        tools/upgrade.sh
请在合并前提交或贮藏您的修改。
正在终止
There was an error updating. Try again later?
```
I don't want to reinstall ohmyzsh, so how to solve the trouble?

**Comment by mcornella** [comment, marked as answer]
You made local modifications to the files reported in the OMZ folder (`cd $ZSH`). You need to check whether you still want them (`git diff`) and either commit them (`git commit -a`) or delete them (`git restore .`). _Note that the git commands I posted need to be run from the OMZ folder_ (`cd $ZSH`).
